template/.gitignore.jinja2

```diff
@@ -32,6 +34,7 @@ src/ontology/tmp/*
 !src/ontology/tmp/README.md
 
 src/ontology/run.sh.conf
+src/ontology/run.sh.env
```

Comment:
Is this to circumvent the mess I made by starting to check in `conf`?

Reply:
No, it’s to use as an argument to Docker’s `--env-file` option. The `run.sh.conf` file may contain variables that are only needed in the `run.sh` script itself and don’t need to be passed to the container (e.g., the `ODK_IMAGE` variable).

But yeah, we also need to take into account that _some people_ commit their `run.sh.conf` files… ^^'

Reply:
Haha thanks!